learning/index.md

```diff
@@ -114,6 +114,43 @@
   <br><br>
 </div>
 
+<br>
+<br>
+<br>
+<!--
+     Containers: Pluto
+-->
+<div class="container pt-sm-2 pluto packages">
+ <br><br>
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-4 col-md-3 language-features"><hr/></div>
+      <div class="col-lg-4 col-md-6 language-features section-heading">
+        <h2 class="lead secondary-heading">Pluto.jl notebooks</h2>
+      </div>
+      <div class="col-lg-4 col-md-3 language-features"><hr/></div>
+    </div>
+
+    <p><a href="https://plutojl.org/">Pluto.jl</a> is a Julia programming environment designed for <em>learning and teaching</em>, and it is the easiest way to get started with Julia programming, packages and visualisation.</p>
```

Comment:
```suggestion
    <p><a href="https://plutojl.org/">Pluto.jl</a> is a Julia programming environment designed for <em>learning and teaching</em>, and it is a great way to get started with Julia programming, packages and visualisation.</p>
```
For large demographics, the easiest way to start with Julia is either pasting the juliaup command or using a replit service, so I'm not comfortable with the unqualified claim of "easiest"